Alignment/include/ActsAlignment/Kernel/detail/AlignmentEngine.hpp

```diff
@@ -118,7 +118,7 @@ TrackAlignmentState trackAlignmentState(
   std::vector<std::pair<size_t, bool>> measurementStates;
   measurementStates.reserve(15);
   // Number of smoothed states on the track
-  size_t nSmoothedStates = 0;
+  size_t nSmoothedStates = 0;  // NOLINT(-Werror,-Wunused-but-set-variable)
```

Comment:
intended unused?

Reply:
hmm does clang-tidy ignore the `[[maybe_unused]]` label?